docs/sas-updates.md

```diff
@@ -1,10 +1,10 @@
-New in SAS Viya platform 2021.2.6: the connect workload class is no longer required. For more information, see [Connect Workload Class Changes](https://go.documentation.sas.com/doc/en/itopscdc/v_026/itopswn/n0jh2fbifqgoksn1uou9p2zgbzdy.htm#p15778dvqwzjtgn1e95nq9v0y1wv).
- 
+New in SAS Viya platform 2021.2.6: the connect workload class is no longer required. For more information, see [Connect Workload Class Changes](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=v_026&docsetId=itopswn&docsetTarget=n0jh2fbifqgoksn1uou9p2zgbzdy.htm#p15778dvqwzjtgn1e95nq9v0y1wv).
+
 To deploy the SAS Viya platform 2021.2.6 and later, use the most recent version of SAS Viya 4 Infrastructure as Code. The default settings do not create a connect node pool. If your current software order has a requirement for the connect node pool, you can use the connect node pool example file in `examples/sample-input-connect.tfvars`.
- 
+
 If you are updating the SAS Viya platform to version 2021.2.6, take some additional steps to remove the connect nodes.
 
-1.	Perform the update by following the steps in the [SAS Viya platform documentation](https://go.documentation.sas.com/doc/en/itopscdc/default/k8sag/p043aa4ghwwom6n1beyfifdgkve7.htm). 
+1.	Perform the update by following the steps in the [SAS Viya platform documentation](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=v_043&docsetId=k8sag&docsetTarget=p043aa4ghwwom6n1beyfifdgkve7.htm).
```

Comment:
Shouldn't it use "cdcVersion=default" instead of "v_043"?

Reply:
In this case, because of the topic, the doc link was intended to point specifically to the 2021.2.6 release instead of "default" which would point to the latest.

Reply:
I asked because v_043 is for 2023.08 which is the latest and the topic is "Update to a New Version".  Is that a reason you wanted to pin the Update instruction to 2023.08? 

The other link, you have it points to v_026 which is 2021.2.6 for the connect workload change, that make sense.

Reply:
I agree that the link on line 7 should not be pinned to 2023.08, I misinterpreted your comment and was thinking of the v_026 link on line 1. I entered the original link on line 7 into the DOC converter tool and it creates a link with "default" in it, not v_043, which appears in the URL after choosing the production link on the redirect page. Updating it to the value with default in it.